app/src/main/resources/CampaignKit.properties

```diff
@@ -1,8 +1,8 @@
 #
 # Download this file to 'CampaignKit.properties' in the root of your project's Java source folder
 #
-CKAPIToken=d310d1e65dfbfb2c449dacb0c74367d8c7a651e217fcab4c05401346748c165c
-CKKitURL=http://campaignkit.radiusnetworks.com/sdk/v1/kits/6
-PKAPIToken=1ca9a9743f368095c2fcce37d19950d251fa3fede404ce2f28734fa109a92680
-PKKitURL=http://proximitykit.radiusnetworks.com/api/kits/3496
-PKAnalyticsURL=https://analytics.radiusnetworks.com/sdk/v1/events
+CKAPIToken=25937d9a7113c2ffedf0ceb9d6956acab078e506e410c72ac5ca7f2f88991e1f
+CKKitURL=http://campaignkit-staging.radiusnetworks.com/sdk/v1/kits/8
```

Comment:
Just switched the CampaignKit.properties file back to what was in there before I started the upgrade.